scalameta/dialects/shared/src/main/scala/scala/meta/Dialect.scala

```diff
@@ -577,14 +577,78 @@ final class Dialect private (
   // Dialects have reference equality semantics,
```

Comment:
Looks like this comment would not make any sense anymore, but I think specially in case of dialects i doesn't help use to compare with reference.

Reply:
good catch, will remove